parmed/openmm/parameters.py

```diff
@@ -116,7 +117,7 @@ def from_parameterset(cls, params, copy=False):
 
         return new_params
 
-    def write(self, dest, provenance=None):
+    def write(self, dest, provenance=None, write_unused=True):
```

Comment:
I like your name better, but not enough to ask @rafwiewiora to open a new PR to fix it.  If he wants to, I'd merge it, but I think this name is acceptable.  So it's up to him.